integration-cli/docker_cli_daemon_test.go

```diff
@@ -1211,15 +1294,14 @@ func (s *DockerDaemonSuite) TestDaemonRestartKillWait(c *check.C) {
 func (s *DockerDaemonSuite) TestHttpsInfo(c *check.C) {
 	const (
 		testDaemonHTTPSAddr = "localhost:4271"
+		host                = "tcp://" + testDaemonHTTPSAddr
```

Comment:
not sure I see the point in having 2 vars when the 1st is only used in the 2nd.  Why not just add "tcp://" to testDaemonHTTPSAddr ?